includes/discovery/fanspeeds/extreme.inc.php

```diff
@@ -13,13 +24,13 @@
         $index = $matches[1];
         // substract 100 from index to start from 1 instead of 101
         $modindex = ($index - 100);
-        $oid = "1.3.6.1.4.1.1916.1.1.1.9.1.4.$index";
-        $value = snmp_get($device, $oid, '-Oqv', 'EXTREME-BASE-MIB');
-        $descr = "Fan Speed $modindex";
-        $high_limit      = ($value * 1.5);
-        $high_warn_limit = ($value * 1.25);
-        $low_warn_limit  = ($value * 0.75);
-        $low_limit       = ($value * 0.5);
+        $oid    = "1.3.6.1.4.1.1916.1.1.1.9.1.4.$index";
```

Comment:
Need to fix the formatting on lines 27 - 33, a lot of additional whitespace and tabs have made it in.